### PR TITLE
Bump msal to 2.0

### PR DESCRIPTION
**Pull request by derisen**
# CHANGELOG

## 7/22/2020

* Updated to MSAL.js 2.0

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -36,7 +36,7 @@ bld/
 vs
 
 # Visual Studio Code cache/options directory
-.vscode/
+.vscode
```
Comment (idg-sam) [inline .gitignore]:
```suggestion
.vscode/
```
You probably want to keep the slash here since its a dir.

File: ProfileSPA/package.json [hunk 2]
```diff
@@ -3,10 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "bootstrap": "^4.4.1",
-    "msal": "^1.3.2",
+    "@azure/msal-browser": "^2.0.0",
+    "bootstrap": "^4.5.0",
+    "msal": "^1.3.3",
```
Comment (idg-sam) [inline ProfileSPA/package.json]: Is the `msal` lib still required, or just `@azure/msal-browser`?

`grep -rnw ./ -e "from 'msal'" ` and `grep -rnw ./ -e 'from \"msal\"'` both yield zero results from this branch.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 7/22/2020
+
+* Updated to MSAL.js 2.0
```
Comment (idg-sam) [inline CHANGELOG.md]:
```suggestion
* Updated to use @azure/msal-browser 2.0
```

File: README.md [hunk 4]
```diff
@@ -9,22 +9,19 @@ products:
 - redux
 - dotnet
 - azure-active-directory
+- msal
```
Comment (idg-sam) [inline README.md:12]:
```suggestion
- msal
- msal-browser
```

**Review by idg-sam — CHANGES_REQUESTED** [review]
Great work. The code looks great. I have noted a couple of minor suggestions. I will give it a test run this afternoon and add more comments if necessary :)

File: ProfileSPA/package.json (hunk 2 above)
Comment (derisen) [reply ProfileSPA/package.json]: ah, good catch!

File: CHANGELOG.md (hunk 3 above)
Comment (derisen) [reply CHANGELOG.md]: slightly changed: * Updated MSAL.js to 2.0 using auth code w/ PKCE.

File: README.md (hunk 4 above)
Comment (derisen) [reply README.md:12]: this section is mostly for ms docs indexing the sample. will keep it "msal" for better results.

**Comment by derisen** [comment]
@idg-sam thanks a lot for the review!

**Pull request merged**